newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['silverspoon'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: 'e85c7b343d3723bfa6d747d3c93d9723f6323746',
```

Comment:
need to ignore this or use environment variable so your license key isn't public